maven-plugin/pom.xml

```diff
@@ -10,7 +10,42 @@
         <version>1.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>maven-plugin</artifactId>
+    <artifactId>sashimono-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+    <name>maven-plugin</name>
+
+    <properties>
+        <maven-core.version>3.9.6</maven-core.version>
+        <maven-plugin-annotations.version>3.12.0</maven-plugin-annotations.version>
+        <junit-jupiter-version>5.10.0</junit-jupiter-version>
+    </properties>
+
+    <dependencies>
+        <dependency>
```

Comment:
It can come in a later PR, but we should move all dependencies up to the parent <dependencyManagement> section.